src/domain/hooks/hooks.repository.ts

```diff
@@ -449,9 +507,12 @@ export class HooksRepositoryWithNotifications implements IHooksRepository {
 @Injectable()
 export class HooksRepository implements IHooksRepository {
   private static readonly HOOK_TYPE = 'hook';
+  private static readonly UNSUPPORTED_CHAIN_EVENT = 'unsupported_chain_event';
```

Comment:
The duplication from here on, as well as in the tests will be solved by https://github.com/safe-global/safe-client-gateway/pull/1928.